### ISSUE TITLE
font.opentype.tables is not enumerable

### ISSUE THREAD

**Issue by darknoon** [Pomax/lib-font#129]
I tried doing `Object.keys(font.opentype.tables)` to get the tables that the font contains but it just returns `[]`

I think this is due to lazy.js
```js
export default function lazy(object, property, getter) {
  let val;
  Object.defineProperty(object, property, {
    get: () => {
      if (val) return val;
      val = getter();
      return val;
    },
  });
}
```

where `Object.defineProperty(…)` has [via MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#description):

> By default, values added using Object.defineProperty() are immutable and not enumerable

I couldn't find much documentation, so it wasn't clear if this is somehow intended. Is there a better way to accomplish this?

**Comment by Pomax** [comment]
Aye, that's by design (because it can cause memory to blow up if it's fully resolved up front). You can use `font.opentype.directory` to get all known tables:

```javascript
  const { directory, tables } = font.opentype;
  const knownTables = directory.map(e => e.tag.trim())
```

**Comment by Pomax** [comment]
we could add enumerable as property option, and see what it does if you want to give that a shot,

**Comment by Pomax** [comment]
landed in 2.4.0
